dev/DynamicDependency/API/MddDetourPackageGraph.cpp

```diff
@@ -120,18 +142,35 @@ HRESULT WINAPI MddDetourPackageGraphInitialize() noexcept
     // Detour package graph APIs to our implementation
     FAIL_FAST_IF_WIN32_ERROR(DetourUpdateThread(GetCurrentThread()));
     FAIL_FAST_IF_WIN32_ERROR(DetourAttach(&(PVOID&)TrueGetCurrentPackageInfo, DynamicGetCurrentPackageInfo));
-    FAIL_FAST_IF_WIN32_ERROR(DetourAttach(&(PVOID&)TrueGetCurrentPackageInfo2, DynamicGetCurrentPackageInfo2));
-    if (IsWindows10_20H1OrGreater())
+    //
+    // NOTE: GetCurrentPackageInfo2 requires >=19H1
+    // NOTE: GetCurrentPackageInfo3 requires >=20H1
```

Comment:
Curious: Why the sliding API window here versus just using lowest common denominator (GetCurrentPackageInfo)? Seems like it would simplify the code below.